packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -329,8 +329,8 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
             spellCheck={ false }
             defaultVisibleValue={ this._currentVisibleValue }
             suggestedDisplayValue={ suggestedDisplayValue }
-            updateValueInWillReceiveProps={ this._onUpdateValueInAutoFillWillReceiveProps }
-            shouldSelectFullInputValueInComponentDidUpdate={ this._onShouldSelectFullInputValueInAutoFillComponentDidUpdate }
+            updateValueInWillReceiveProps={ this._onUpdateValueInAutofillWillReceiveProps }
+            shouldSelectFullInputValueInComponentDidUpdate={ this._onShouldSelectFullInputValueInAutofillComponentDidUpdate }
```

Comment:
Is there a shorter name that can convey the purpose of this variable?

Reply:
Maybe, I don't remember when this got added. I don't mind long names, they get compiled down!